src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Colors color;
+    private final int number;
+
+    public Ball(Colors color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball{" + "COLOR=" + color + ", number=" + number + '}';

```

Comment:
It is a good practice to use the next format to implement toString method:
```suggestion
        return "Ball{" 
            + "color=" + color 
            + ", number=" + number 
            + '}';
```